packages/api/src/codegen/factory.ts

```diff
@@ -3,7 +3,9 @@ import type Oas from 'oas';
 
 import TSGenerator from './languages/typescript/index.js';
 
-export type SupportedLanguages = 'js' | 'js-cjs' | 'js-esm' | 'ts';
+export enum SupportedLanguages {
```

Comment:
not a huge deal but i've seen a few TS folks online express their disdain for enums:

https://www.youtube.com/watch?v=jjMbPt_H3RQ
https://www.youtube.com/watch?v=vo6C0RwagyY

Reply:
I don't love them but since this is only used internally I think it helps to clean up how we reference these supported languages.